saleor/graphql/product/bulk_mutations/products.py

```diff
@@ -94,14 +94,21 @@ def perform_mutation(cls, _root, info, ids, **data):
                 variant__in=variants, order__status=OrderStatus.DRAFT
             ).values_list("pk", flat=True)
         )
-
+        data["manager"] = info.context.plugins
```

Comment:
Why not:
`response = super().perform_mutation(_root, info, ids, manager=info.context.plugins, **data)`